app/models/journeys/Journey.scala

```diff
@@ -313,6 +313,11 @@ object Journey extends Enum[Journey] with utils.PlayJsonEnum[Journey] {
   sealed abstract class AdjustmentsBaseJourney(override val entryName: String) extends Journey(entryName) {
     override def toString: String = entryName
   }
+
+  case object Adjustments extends AdjustmentsBaseJourney("adjustments-work-out-profit-and-loss") {
+    override val pageKeys: List[PageName] = List.empty
```

Comment:
Sorry thought this was the base journey but it isn't. We don't need the case object Adjustments because we have profit and loss below, so this object can just be deleted

Reply:
Done :)